core/src/services/redis/backend.rs

```diff
@@ -281,7 +281,7 @@ impl RedisBuilder {
                 let port = ep_url.port_u16().unwrap_or(DEFAULT_REDIS_PORT);
                 ConnectionAddr::Tcp(host, port)
             }
-            Some("rediss") => {
+            Some("redis") => {
```

Comment:
Hi, this is not typo, rediss means that redis over tls, like https, https://www.iana.org/assignments/uri-schemes/prov/rediss. Please revert it.

Reply:
thank you